tools/experimental-workspace/src/recordings/metadata/displayForms/label.owner.region/elements.json

```diff
@@ -0,0 +1,10 @@
+[
+    {
+        "title": "East Coast",
+        "uri": "/gdc/md/l32xdyl4bjuzgf9kkqr2avl55gtuyjwf/obj/1086/elements?id=460488"
```

Comment:
This uri is from a different project than specified in refresh-ldm.sh and refresh-recordings.sh